### PR TITLE
Fixed typo, adjust one expression

### PR DESCRIPTION
**Pull request by flopanzer**
(my first commit here. Hope I'm correct :) )

### CODE REVIEW

File: content/resources/Playground_aspects/External_PG_assets.md [hunk 1]
```diff
@@ -17,17 +17,24 @@ Sometimes you might want to load you own assets into the playground. Should the
  ```javascript
 myGithubUserName/myRepository/commit/2bd79648e08709145cd9575e6679b2ea360f12f6
```
Comment (deltakosh) [inline content/resources/Playground_aspects/External_PG_assets.md:18]: This line should be removed

File: content/resources/Playground_aspects/External_PG_assets.md [hunk 2]
```diff
@@ -17,17 +17,24 @@ Sometimes you might want to load you own assets into the playground. Should the
...
  ```
- change the /commit/ part to @
+ replace '/commit/' with '@'
 
  ```javascript
 myGithubUserName/myRepository@2bd79648e08709145cd9575e6679b2ea360f12f6
```
Comment (deltakosh) [inline content/resources/Playground_aspects/External_PG_assets.md:23]: same

File: content/resources/Playground_aspects/External_PG_assets.md (hunk 1 above)
Comment (flopanzer) [reply content/resources/Playground_aspects/External_PG_assets.md:18]: For me as a beginner a random commit - hash is more self explanatory than something like: 
'myGithubUserName/myRepository/commit/hash'

Comment (flopanzer) [reply content/resources/Playground_aspects/External_PG_assets.md:18]:
```suggestion
myGithubUserName/myRepository/commit/hash
```

File: content/resources/Playground_aspects/External_PG_assets.md (hunk 2 above)
Comment (flopanzer) [reply content/resources/Playground_aspects/External_PG_assets.md:23]:
```suggestion
myGithubUserName/myRepository@hash
```

Comment (deltakosh) [reply content/resources/Playground_aspects/External_PG_assets.md]: Well my problem is that the entire line is not supposed to be in the .md file ;)

File: content/resources/Playground_aspects/External_PG_assets.md (hunk 1 above)
Comment (deltakosh) [reply content/resources/Playground_aspects/External_PG_assets.md:18]: ok you are right! let's keep it then 
sorry about the change

Comment (flopanzer) [reply content/resources/Playground_aspects/External_PG_assets.md:18]: No worries, I was just a little confused what you wanted :)

**Review by deltakosh — APPROVED** [review]

**Pull request merged**